src/interfaces.ts

```diff
@@ -144,7 +144,7 @@ export type WithChangeLog = {
 /**
  * @hidden Developers should use [[unstable_getResourceAcl]] and [[unstable_getFallbackAcl]] to access these.
  */
-export type unstable_Acl = {
+export type unstable_WithAcl = {
   acl: {
     resourceAcl?: unstable_AclDataset;
```

Comment:
I saw you changed this to use `null` instead of `undefined` elsewhere? I initially did it this way because it's likely that the Resource ACL does not exist, whereas the fallback ACL should always exist - but since it's perfectly possible that the fallback ACL is not accessible to the current user, this definitely also makes more sense:

```suggestion
    resourceAcl: unstable_AclDataset | null;
```